acme.sh

```diff
@@ -1999,7 +1999,13 @@ _post() {
     if [ "$_ret" != "0" ]; then
       _err "Please refer to https://www.gnu.org/software/wget/manual/html_node/Exit-Status.html for error code: $_ret"
     fi
-    _sed_i "s/^ *//g" "$HTTP_HEADER"
+    if echo $_WGET | grep -q " -d " 2>/dev/null; then
```

Comment:
use `if _contains "$_WGET" " -d " `

Reply:
_contains doesn't pass the "-q" parameter. So it may throw an error instead of just providing an exit code. Won't it?

From the man pages:
```
       -q, --quiet, --silent
              Quiet; do not write anything to standard output.  Exit immediately with zero status  if  any  match  is
              found, even if an error was detected.  Also see the -s or --no-messages option.
```